_stbt/mask.py

```diff
@@ -180,17 +180,29 @@ def __repr__(self):
             return f"{prefix}{self._region!r}"
 
     def __add__(self, other: MaskTypes) -> "Mask":
-        if isinstance(other, (Region, Mask)):
+        if isinstance(other, (Region, Mask, type(None))):
```

Comment:
None as the empty region really shows its ugliness here.

This might be clearer:

```suggestion
        if other is None or isinstance(other, (Region, Mask)):
```